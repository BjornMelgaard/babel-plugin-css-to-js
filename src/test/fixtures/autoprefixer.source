
css`.foo { transform: ${ foo }; color: ${ bar } }`;
