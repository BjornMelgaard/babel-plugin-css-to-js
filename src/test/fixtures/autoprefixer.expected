
({
  ".foo": {
    webkitTransform: foo,
    transform: foo,
    color: bar
  }
});